packages/terra-button/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Updated test pages with icons to use a `terra-icon`
+
+### Fixed
+* Fixed the Compact Button test page not displaying a compact button
```

Comment:
nit: Fixed the Compact Button test page to display a compact button
     
IMO I find it easier to understand a change log when it describe what changes verse trying to decide what was wrong and needed a change